GRPC/grpcServer/grpcserver.go

```diff
@@ -0,0 +1,46 @@
+package grpcServer
+
+import (
+	"context"
+	"fmt"
+	courierProto "github.com/Baraulia/COURIER_SERVICE/GRPC"
+	"github.com/Baraulia/COURIER_SERVICE/service"
+	"github.com/sirupsen/logrus"
```

Comment:
we decided that we would use package "log" in COURIER_SERVICE. I'm not sure if it's worth using?